packages/react/src/patterns/sub-patterns/ContentSection/ContentSection.js

```diff
@@ -22,41 +22,47 @@ const { prefix } = settings;
  * @param {string} props.theme theme name
  * @param {*} props.children JSX Components
  * @param {string} props.customClassName optional class to be applied to the containing node
+ * @param {object} props.ref object
```

Comment:
I don't think this change should be necessary. See comment above.

Reply:
okay